src/lib/fixedDecimal.js

```diff
@@ -0,0 +1,106 @@
+"use strict"
+;(function (GLOBAL) {
+
+let Big,
+DP = 20,
+RM = 1,
+P = {},
+UNDEFINED = void 0;
+
+function _Big_() {
+
+        function parse(x, n) {
+            x.number = Number(parseFloat(n));
+        }
+
+        function Big(n) {
+            var x = this;
+
+            if (!(x instanceof Big)) return n === UNDEFINED ? _Big_() : new Big(n);
+
+            if (n instanceof Big) {
+                x.number = n.number;
+            } else {
+                parse(x, n);
+            }
+
+            x.constructor = Big;
+        }
+        Big.prototype = P;
+        Big.DP = DP;
+        Big.RM = RM;
+        return Big;
+    }
+
+    P.round = function(dp,rm) {
+        let Big = this.constructor;
+        let returnValue = new Big(this);
+        returnValue.number = Number(parseFloat(this.number).toFixed(dp));
+        return returnValue;
+    };
+
+    P.add = function(number) {
+        let Big = this.constructor;
+        let returnValue = new Big(this);
+        returnValue.number =  Number(parseFloat(this.number)+number).toFixed(Big.DP);
```

Comment:
seems closing parentheses seems to be at the wrong place here, .toFixed returns string:
```
Number(parseFloat(this.number)+number).toFixed(Big.DP)
```
maybe that's why you needed to hack with parseFloats ?